AEPNotificationContent/Sources/Model/TimerPayload.swift

```diff
@@ -0,0 +1,79 @@
+/*
+ Copyright 2024 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+import UIKit
+import UserNotifications
+
+class TimerPayload: Payload {
+    // MARK: - Properties
+
+    /// The image URL for the timer expired view
+    var alternateImageURL: URL
+
+    /// The color of the timer text
+    var timerColor: UIColor
+
+    /// The duration of the timer
+    var expiryTime: TimeInterval
+
+    /// titleBody of non-expired timer template view
+    lazy var titleBodyPayload: TitleBodyPayload = {
+        let title = notificationContent.expandedTitle ?? notificationContent.title
+        let body = expandedBody ?? ""
+        return TitleBodyPayload(title: title, body: body)
+    }()
+
+    /// titleBody for expired timer template view
+    lazy var altTitleBodyPayload: TitleBodyPayload = {
+        let title = notificationContent.alternateTitle ?? notificationContent.title
+        let body = notificationContent.alternateBody ?? ""
+        return TitleBodyPayload(title: title, body: body)
+    }()
+
+    /// Initializes a `TimerPayload` instance from a `UNNotificationContent`.
+    /// Initialization fails if the mandatory properties required for TimerTemplate are unavailable
+    /// - Parameter notification: The content of the notification.
+    required init?(from notificationContent: UNNotificationContent, notificationDate: Date) {
+        // Extract the alternate image data and fast fail if alternateImage URL is not available
+        guard let alternateImageURL = notificationContent.alternateImageURL else {
```

Comment:
don't think this should be required

Reply:
I will wait until we discuss it later this afternoon to make any change